algorithms-and-data-structures/basic-javascript/index010.js

```diff
@@ -0,0 +1,3 @@
+var xp = 0;
+
+// To keep track of the players expereince points, we've declared a variable called xp and assigned it the starting value of 0. Create another variable to keep track of health and start it at 100.
```

Comment:
I would new line each new sentence as this will help with reading it on a smaller screen (with or without word wrap enabled). Also spelling `expereince` -> `experience`
```suggestion
// To keep track of the players experience points, we've declared a variable called xp and assigned it the starting value of 0. 
// Create another variable to keep track of health and start it at 100.
```